README.md

```diff
@@ -53,7 +53,12 @@ func main() {
         log.Fatalf(err.Error())
     }
 
-    client := opsmngr.NewClient(tc)
+    clientops := opsmngr.SetBaseURL("https://opsmanagerurl/api/public/v1.0/")
+	client, err := opsmngr.New(tc, clientops)
+	if err != nil {
+		log.Fatalf(err.Error())
+	}
```

Comment:
you may want to adjust the indentation here as right now it has some extra spaces